web_timeline/static/lib/vis/vis.css

```diff
@@ -0,0 +1,810 @@
+.vis .overlay {
```

Comment:
I know the benefits of npm (versioning and dependencies management) but my concern is to not introduce an additional skill requirement to be able to install this addon in odoo...